src/test/java/org/candlepin/resource/ConsumerResourceTest.java

```diff
@@ -451,6 +451,27 @@ protected Owner createOwner(String key) {
         return owner;
     }
 
+    protected Owner createOwnerEnt() {
+        return createOwnerEnt(null);
+    }
+
+    protected Owner createOwnerEnt(String key) {
```

Comment:
Better naming and/or documentation would go a long way here to avoiding confusion. It's not clear until you read the method's logic if we're creating an entitlement for an owner, an owner with an entitlement, or an owner in entitlement mode.

e.g. `createEntitlementModeOwner`, `createNonSCAOwner`, `createLegacyCAOwner`

Reply:
done